mobilenet-burn/src/model/inverted_residual.rs

```diff
@@ -0,0 +1,94 @@
+use super::conv_norm::NormalizationLayer;
+use super::conv_norm::NormalizationType;
+use super::conv_norm::{Conv2dNormActivation, Conv2dNormActivationConfig};
+use alloc::vec;
+use alloc::vec::Vec;
+use burn::config::Config;
+use burn::nn::conv::Conv2dConfig;
+use burn::nn::BatchNormConfig;
+use burn::tensor::Tensor;
+use burn::{module::Module, nn::conv::Conv2d, tensor::backend::Backend};
+#[derive(Module, Debug)]
+enum InvertedResidualSequentialType<B: Backend> {
+    Conv(Conv2d<B>),
+    ConvNormActivation(Conv2dNormActivation<B>),
+    NormLayer(NormalizationLayer<B, 4>),
+}
+
+#[derive(Module, Debug)]
+pub struct InvertedResidual<B: Backend> {
+    use_res_connect: bool,
+    layers: Vec<InvertedResidualSequentialType<B>>,
+    out_channels: usize,
+    is_cn: bool,
+}
```

Comment:
I'd add a small docstring, could simple point to a reference (e.g, https://paperswithcode.com/method/inverted-residual-block)